curriculum/scripts/roadmap.py

```
@@ -0,0 +1,366 @@
+
+    courses = load_catalog()
+    budget = args.weeks * args.hours_per_week
+
+    if args.show_prompt:
+        print(build_prompt(courses, args.goal, budget, args.weeks,
+                           args.hours_per_week, args.level))
+        print("=" * 70)
+
+    print(f"카탈로그 {len(courses)}개 / 모델 {args.model}")
+    print(f"목표: {args.goal}")
+    print(f"예산: {args.weeks}주 × {args.hours_per_week}h = {budget}h · 수준 L{args.level} ({LEVEL_LABEL[args.level]})")
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)